package com.example.myapplication

import android.app.Application
import dagger.hilt.android.HiltAndroidApp

@HiltAndroidApp(Application::class)
class CalendarPlannerApplication : Hilt_CalendarPlannerApplication()
